### DISCUSSION TITLE
How to help translation?

### DISCUSSION THREAD

**Discussion by Thecode764** [docsifyjs/docsify#2463] [Q&A] [answered]
hi i want to help translate docsify docs to persian. how?

**Comment by Koooooo-7** [comment]
Hi @Thecode764 , thx a lot for your contribution proposal!
Currently we are facing a problem that it is hard to sync the translation docs up to date once the main contributor inactive.
So we are still working on find a feasible way to provide a good experience for users now.
Getting a new translation is not on the table for now, but you could make your own one for this.
We may find a place to list the awesome translation docs for sharing some i18n works for your job!

thx again!

**Comment by sy-records** [comment, marked as answer]
Our new version will use the crowdin system, which you can translate here.

I added the persian language. https://translate.docsifyjs.org/project/docsify/fa#/develop
